nav/stepper/stepper_constants.py

```diff
@@ -1,26 +1,28 @@
+import math
 # Stepper mode select
 STEPPER_m0 = 17
 STEPPER_m1 = 27
 STEPPER_m2 = 22
 
 # First wheel GPIO Pin
-WHEEL1_step = 21
-WHEEL1_dir = 20
+BR_step = 21
+BR_dir = 20
 
 # Second wheel GPIO Pin
-WHEEL2_step = 11
-WHEEL2_dir = 10
+FR_step = 11
+FR_dir = 10
 
 # Third wheel GPIO Pin
-WHEEL3_step = 7
-WHEEL3_dir = 8
+BL_step = 7
+BL_dir = 8
 
 # Fourth wheel GPIO Pin
-WHEEL4_step = 24
-WHEEL4_dir = 23
+FL_step = 24
+FL_dir = 23
 
 CW = 1 			                 # Clockwise rotation
 CCW = 0 		                 # Counter-clockwise rotation
 STEPS_PER_REVOLUTION = 800 		 # Steps per Revolution (360/1.8) * 4. Multiply by 4 because quarter step.
-
+DISTANCE = 60*math.pi/25.4
```

Comment:
I feel like this and steps_per_revolution will be different for different wheels. All these values should be in the same place so they are easier to adjust in the future.

Reply:
DISTANCE will change based on wheel size, steps_per_revolution will only change if we change the step mode.